sync_volunteer_data/frontend/sync-galaxy-digital-data.js

```diff
@@ -0,0 +1,125 @@
+// Script used to convert Volunteer data from the
+// (Galaxy Digital)[http://api2.galaxydigital.com/volunteer/docs/] APIs
+// into the AirTable schemas
+import { globalConfig } from "@airtable/blocks";
+
+const AIRTABLE_MAX = 10;
+const GALAXY_DIGITAL_MAX = 50;
+
+async function getGalaxyDigitalVolunteerData(limit, offset) {
+  const GALAXY_DIGITAL_API_KEY = globalConfig.get("GALAXY_DIGITAL_API_KEY");
+  const url = new URL("volunteer/user/list/", "https://api2.galaxydigital.com");
+  const params = url.searchParams;
+  params.set("key", GALAXY_DIGITAL_API_KEY);
+  params.set("limit", limit);
+  params.set("offset", offset);
+
+  let response = await fetch(url, { method: "GET", mode: "no-cors" });
+  let data = await response.json();
```

Comment:
@alienngator I'm getting an exception thrown here because of invalid JSON, but the response returns valid data so I'm totally lost, any ideas?